spec/fixtures/modules/results/plans/test_methods.pp

```diff
@@ -0,0 +1,15 @@
+plan results::test_methods(
+  String $target,
+  Optional[String] $fail = 'false'
```

Comment:
Did you add $fail for a future test? And is the hashkey 'fail' an instrumentation for testing purposes?  Or does it serve a purpose other than testing?